src/child/actions/window.js

```diff
@@ -4,17 +4,18 @@ import createActionCreator from '../utils/createActionCreator';
 import currentWindowService from '../services/currentWindowService';
 
 export const minimise = createActionCreator(() => ({
-    type: ACTION_TYPES.MINIMIZE
+    type: ACTION_TYPES.MINIMISE
```

Comment:
Renaming for consistency is a great idea 👍  however, I think I'd rename to be consistent with the APIs which we're using (i.e. using the American spelling which OpenFin and the `Window` api use) to reduce any confusion about which spelling to use.